src/components/KakaoButton/index.tsx

```diff
@@ -18,8 +20,8 @@ const KakaoButton: FC<KakaoButtonProps> = ({ url }: KakaoButtonProps) => {
     Kakao.Link.sendDefault({
       objectType: 'feed',
       content: {
-        title: '무비로그',
-        description: '공유할끄니까',
+        title: { title },
```

Comment:
```suggestion
        title,
```
로 해도 오브젝트에 똑같이 들어가서 좀더 축약해서 표현할 수 있어!
https://alligator.io/js/object-property-shorthand-es6/
여기 참고하면 도움될거야. 아래도 똑같이 수정하면 될듯!